src/App.php

```diff
@@ -387,6 +387,54 @@ private function loadConfigFiles()
 				$this->setConfigValue('system', 'language', $lang);
 				unset($lang);
 			}
+
+			if (isset($friendica_platform)) {
+				$this->setConfigValue('system', 'friendica_platform', $friendica_platform);
+				unset($friendica_platform);
+			} else {
+				// Default value until 2018.12-dev
+				$this->setConfigValue('system', 'friendica_platform', 'Friendica');
+			}
+
+			if (isset($friendica_codename)) {
+				$this->setConfigValue('system', 'friendica_codename', $friendica_codename);
+				unset($friendica_codename);
+			} else {
+				// Default value until 2018.12-dev
+				$this->setConfigValue('system', 'friendica_codename', 'The Tazmans Flax-lily');
+			}
+
+			if (isset($friendica_version)) {
+				$this->setConfigValue('system', 'friendica_version', $friendica_version);
+				unset($friendica_version);
+			} else {
+				// Default value until 2018.12-dev
+				$this->setConfigValue('system', 'friendica_version', '2018.12-dev');
+			}
+
+			if (isset($dfrn_protocol_version)) {
+				$this->setConfigValue('system', 'dfrn_protocol_version', $dfrn_protocol_version);
+				unset($dfrn_protocol_version);
+			} else {
+				// Default value until 2018.12-dev
+				$this->setConfigValue('system', 'dfrn_protocol_version', '2.23');
+			}
+
+			if (isset($db_update_version)) {
+				$this->setConfigValue('system', '$db_update_version', $db_update_version);
+				unset($db_update_version);
+			} else {
+				// Default value until 2018.12-dev
+				$this->setConfigValue('system', 'db_update_version', '1284');
```

Comment:
I don't know if this is a good idea.. Shall we let just this define in `boot.php` for backward compatibility (just in case no `config.ini.php` is found)

Reply:
I don't even understand where the variables you're checking for are coming from?